san/sanbase_graphql.py

```diff
@@ -640,3 +615,10 @@ def _format_to_date(datetime_obj_or_str):
     dt = iso8601.parse_date(datetime_obj_or_str) + \
         datetime.timedelta(hours=23, minutes=59, seconds=59)
     return dt.isoformat()
+
+def _format_return_fields(list_to_be_modified, list_to_be_iterated):
```

Comment:
we can use `map` - this eliminates need for `list_to_be_modified`

Reply:
You're right, it does optimise things a bit